app/qml/map/MMRecordingTools.qml

```diff
@@ -51,11 +51,11 @@ Item {
     target: mapTool
 
     function onFinalEmptyGeometry() {
-      __notificationModel.addWarning( "The feature has empty geometry, after avoiding overlaps!" )
+      __notificationModel.addWarning( qsTr( "The feature has empty geometry, after avoiding overlaps! Try drawing new one after clicking \"Edit Geometry\"." ) )
```

Comment:
```suggestion
      __notificationModel.addError( qsTr( "Geometry was removed to avoid overlaps with other features. Tap \"Edit Geometry\" to draw a new one.) )
```